hawc_hal/HAL.py

```diff
@@ -265,34 +267,34 @@ def display(self, verbose=False):
         Prints summary of the current object content.
         """
 
-        print("Region of Interest: ")
-        print("-------------------\n")
+        log.info("Region of Interest: ")
+        log.info("-------------------\n")
         self._roi.display()
```

Comment:
Only a style call :) but do you want the return chars there?

Reply:
I don't really care, I just copied it from the original 😆 I can remove the \n.